controller/PciManager.php

```diff
@@ -252,7 +252,7 @@ protected function getMinifiedModel(PortableElementObject $object)
     }
 
     /**
-     * @return |null
+     * @return null
```

Comment:
I see the `return null;` has been removed, so we can change the `@return`. I also noticed that the `$pciDataObject` that get's returned is of the type `\oat\qtiItemPci\model\portableElement\dataObject\PciDataObject`. So I would suggest importing that class, and changing this to `@return PciDataObject`